File: android_app/simple_android_project/src/main/java/com/smartagent/technician/MainActivity.java
package com.smartagent.technician;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;
import android.widget.LinearLayout;
import android.view.Gravity;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        
        // Create layout programmatically (no XML needed)
        LinearLayout layout = new LinearLayout(this);
        layout.setOrientation(LinearLayout.VERTICAL);
        layout.setGravity(Gravity.CENTER);
        
        TextView title = new TextView(this);
        title.setText("🔧 SmartAgent טכנאי");
        title.setTextSize(24);
        title.setGravity(Gravity.CENTER);
        
        TextView subtitle = new TextView(this);
        subtitle.setText("אפליקציית טכנאי חכם");
        subtitle.setTextSize(16);
        subtitle.setGravity(Gravity.CENTER);
        
        layout.addView(title);
        layout.addView(subtitle);
        setContentView(layout);
    }
}

File: simple_app/app/src/main/java/com/smartagent/simple/MainActivity.kt
package com.smartagent.simple

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        
        val textView = TextView(this)
        textView.text = "🚀 SmartAgent - האפליקציה לטכנאים חכמים"
        textView.textSize = 24f
        
        setContentView(textView)
    }
}
            setTextColor(Color.GRAY)
            setPadding(0, 0, 0, 60)
        }
        
        val startButton = Button(this).apply {
            text = "התחל עבודה"
            textSize = 16f
            setPadding(20, 20, 20, 20)
            layoutParams = ViewGroup.LayoutParams(
                ViewGroup.LayoutParams.MATCH_PARENT,
                ViewGroup.LayoutParams.WRAP_CONTENT
            )
            setOnClickListener {
                // כאן יהיה הלוגיק של התחלת עבודה
            }
        }
        
        val settingsButton = Button(this).apply {
            text = "הגדרות"
            textSize = 16f
            setPadding(20, 20, 20, 20)
            layoutParams = ViewGroup.LayoutParams(
                ViewGroup.LayoutParams.MATCH_PARENT,
                ViewGroup.LayoutParams.WRAP_CONTENT
            )
            setOnClickListener {
                // כאן יהיה הלוגיק של הגדרות
            }
        }
        
        layout.addView(title)
        layout.addView(subtitle)
        layout.addView(startButton)
        layout.addView(settingsButton)
        
        setContentView(layout)
    }
}
